Tests/MongoSwiftSyncTests/SpecTestRunner/FailPoint.swift

```diff
@@ -166,6 +166,8 @@ internal struct FailPoint: Decodable {
     public static func failCommand(
         failCommands: [String],
         mode: Mode,
+        blockConnection: Bool? = nil,
+        blockTimeMS: Int? = nil,
```

Comment:
Given that we'll always want `blockConnection` to be true if we've specified a non-nil `blockTimeMS`, can we drop the `blockConnection` argument to this method and just set it true if `blockTimeMS` is specified?